jenkinsfiles/build_win_go.groovy

```diff
@@ -45,17 +45,17 @@ pipeline {
             }
         }
 
-        // stage ('Run test samples') {
-        //     steps {
-        //         withCredentials([string(credentialsId: 'jenkins/core-sdk-key', variable: 'ENV_LICENSE_KEY')]) {
-        //             dir('build/PDFTronGo/pdftron/samples') {
-        //                 sh '''
-        //                     ./runall_go.bat
-        //                 '''
-        //             }
-        //         }
-        //     }
-        // }
+        stage ('Run test samples') {
+            steps {
+                withCredentials([string(credentialsId: 'jenkins/core-sdk-key', variable: 'ENV_LICENSE_KEY')]) {
+                    dir('build/PDFTronGo/pdftron/samples') {
+                        sh '''
+                            ./runall_go.bat
+                        '''
+                    }
+                }
+            }
+        }
```

Comment:
You should talk with Ramzi regarding this.

Reply:
We can leave this commented out, I've added a task to just go through the couple of tests that are failing here.

Reply:
will update